src/scripts/notify.sh

```diff
@@ -105,8 +105,8 @@ FilterBy() {
         fi
     done
     # If the invert_match parameter is set, invert the match.
-    if [ "$FLAG_MATCHES_FILTER" = "false" ] && [ "$SLACK_PARAM_INVERT_MATCH" -eq 0 ] ||
-       [ "$FLAG_MATCHES_FILTER" = "true" ] && [ "$SLACK_PARAM_INVERT_MATCH" -eq 1 ]
+    if { [ "$FLAG_MATCHES_FILTER" = "false" ] && [ "$SLACK_PARAM_INVERT_MATCH" -eq 0 ]; } || \
```

Comment:
This could probably be done better. I believe "$FLAG_MATCHES_FILTER" is a number before it gets quoted, if so, something like this could be performed:
 
(psuedo code)

```
match = $FLAG_MATCHES_FILTER

if [ invert match ]; then
    match = $(1 - $FLAG_MATCHES_FILTER) # invert match boolean
fi
```